client/my-sites/site-settings/settings-reading/main.tsx

```diff
@@ -1,14 +1,56 @@
 import config from '@automattic/calypso-config';
 import { useTranslate } from 'i18n-calypso';
+import { pick } from 'lodash';
 import DocumentHead from 'calypso/components/data/document-head';
 import FormattedHeader from 'calypso/components/formatted-header';
 import Main from 'calypso/components/main';
+import { SiteSettingsSection } from '../reading-site-settings';
+import wrapSettingsForm from '../wrap-settings-form';
 import { NewsletterSettingsSection } from './newsletter-settings-section';
 import { RssFeedSettingsSection } from './rss-feed-settings-section';
-import { SiteSettingsSection } from './site-settings-section';
 
 const isEnabled = config.isEnabled( 'settings/modernize-reading-settings' );
 
+// Settings are not typed yet, so we need to use `unknown` for now.
+type Settings = unknown;
+
+const getFormSettings = ( settings: Settings ) => pick( settings, [ 'posts_per_page' ] );
```

Comment:
For a couple of moments I was thinking about replacing lodash's `pick` with destructuring, but the result wasn't that elegant as when `pick` is used; plus we are already relying on it in other settings sections' source.

Reply:
Yeah, either will work. 😃

Reply:
I think there's a movement to reduce the usage of Lodash over time. p4TIVU-9Bf-p2
I wonder if there are other places in Calypso that has replaced `pick` with something else, if not with destructuring.

Reply:
> I think there's a movement to reduce the usage of Lodash over time.

In that case I have replaced the usage of the pick method with es6 solution.